### ISSUE TITLE
Is this complete ? Can i have the documentation or something like readme

### ISSUE THREAD

**Issue by flouthoc** [yorickdewid/dbtree#5]
hey i need documentaion or some sort of manual if it is complete . :smile:  Thanks

**Comment by yorickdewid** [comment]
Hey,

The btree works and the database _can_ be used, but has some complications regarding datatypes. I'm not actively working on this project anymore, but you can checkout the successor [Quantica](https://github.com/yorickdewid/Quantica) which is far more advanced.

If you still want to use this project somehow, please let me know and I'll write a README

Cheers

**Comment by flouthoc** [comment]
Yes i have already seen Quantica looks great to me , but i wanted to study btree that's why i thought it would be much better to read a project that is using btree as a standard abstract data type and dbtree is exaclty what i need i guess , small readme would work for me i guess  otherwise i can work with code too , looks quite documented to me . Thanks

Cheers

**Comment by yorickdewid** [comment]
There is an readme in the source directory which explains the structure. I'm not sure if this will provide a good example for the btree. I'm planning on writing an article about tree structures and sorting algorithms. I'll update you when ready
